### PR TITLE
Allow disabling embedded Bitcode

### PR DESCRIPTION
**Pull request by bithug**
Just an option added to allow shrinking the fat lib size if no bitcode needed (in my case ~37MB --> 13.4MB).
Notice: I care only with version 1.0.2 and lower

Thank you,
bithug

### CODE REVIEW

**Comment by bithug** [comment]
Hm it seems travis just timed out after 46 min 7 sec?
'The job exceeded the maximum time limit for jobs, and has been terminated.'

**Comment by x2on** [comment]
I restarted the build

**Comment by oneton** [comment]
I personally think that this option should be universal instead of only for 1.0.2.

**Comment by x2on** [comment]
@oneton fully agree, didn't get that, thanks for your hint.

@bithug could you please support this option also for 1.1.x?

File: build-libssl.sh [hunk 1]
```diff
@@ -57,6 +57,7 @@ echo_help()
   echo "Options for OpenSSL 1.0.2 and lower ONLY"
   echo "     --archs=\"ARCH ARCH ...\"       Space-separated list of architectures to build"
   echo "                                     Options: ${DEFAULTARCHS}"
+  echo "     --disable-bitcode             Disable embedding Bitcode"
```
Comment (x2on) [inline build-libssl.sh]: Please also support 1.1.0+

Comment (bithug) [inline build-libssl.sh]: Yeah thats a reasonable request, I just didn't need that and had no spare time to figure it out how it works.
I'm not a perl expert so I hope there are no excuses about the changes in the config file :)

File: config/20-ios-tvos-cross.conf [hunk 2]
```diff
@@ -9,7 +9,7 @@
     "ios-tvos-cross-base" => {
         template         => 1,
         cflags           => combine('-isysroot $(CROSS_TOP)/SDKs/$(CROSS_SDK) -fno-common',
-                				sub { (defined($ENV{'SDKVERSION'}) && $ENV{'SDKVERSION'} =~ /^(9|[1-9][0-9]+)\./ && $disabled{shared})
+                				sub { ((!defined($ENV{'CONFIG_DISABLE_BITCODE'}) || $ENV{'CONFIG_DISABLE_BITCODE'} !~ /true/) && defined($ENV{'SDKVERSION'}) && $ENV{'SDKVERSION'} =~ /^(9|[1-9][0-9]+)\./ && $disabled{shared})
```
Comment (oneton) [inline config/20-ios-tvos-cross.conf]: One minor remark. I'd prefer a string comparison over the regex compare for CONFIG_DISABLE_BITCODE. I had to look it up, but changing the comparison to `$ENV{'CONFIG_DISABLE_BITCODE'} ne 'true'` will do the trick.

Comment (bithug) [inline config/20-ios-tvos-cross.conf]: Great, thanks! Just checked it with comparing with ne 'true' and seems working fine for both cases.

**Pull request merged**